package.json

```diff
@@ -14,19 +14,19 @@
     },
     "dependencies": {
         "ajv": "^6.10.0",
-        "bootstrap": "^4.3.1",
-        "node-sass": "^4.13.1",
-        "react": "next",
-        "react-big-calendar": "^0.24.5",
+        "bootstrap": "^4.5.0",
+        "node-sass": "^4.14.1",
+        "react": "0.0.0-33c3af284",
```

Comment:
Do we know why this and `"react-dom"` have been changed? It used to be `"next"` for both.

Reply:
I think running `npm update` set the version to the current upcoming version. I'll change it back to next.